debian/chromium-browser.sh.in

```diff
@@ -7,6 +7,13 @@
 #  Fabien Tassin <fta@sofaraway.org>
 # License: GPLv2 or later
 
+ABSOLUTE_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/$(basename "${BASH_SOURCE[0]}")"
+
+if ! eos-parental-controls-client check ${ABSOLUTE_PATH} >/dev/null; then
```

Comment:
@pwithnall looks like the malcontent-tools package can't be installed as is now and I was wondering if we should use eos-parental-controls here and then update once malcontent package is fixed. What do you think?

Reply:
What error do you get?

As mentioned in one of the tasks, I think we should just bite the bullet and transition to malcontent everywhere now, rather than making more transition work for ourselves in the near future.

Reply:
The malcontent-tools package is not installable:
```
The following packages have unmet dependencies:
 malcontent-tools : Depends: malcontent-data (= 0.3.0-0bem1) but it is not going to be installed
                    Depends: gir1.2-malcontent-0 but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

And when I try to manually install malcontent-data it tries to uninstall a bunch of packages:
```
The following packages will be REMOVED:
  eos-boot-helper eos-browser-tools eos-core eos-core-amd64 eos-google-chrome-helper eos-install-app-helper eos-kalite-tools eos-metrics-instrumentation eos-parental-controls-data eos-updater eos-updater-tools
  flatpak flatpak-builder gdm3 gir1.2-eos-parental-controls-0 gir1.2-eos-updater-0 gir1.2-flatpak-1.0 gnome-control-center gnome-session gnome-shell gnome-software gnome-software-plugin-flatpak
  libeos-parental-controls-0 libflatpak0 python3-flapjack
```

Reply:
Right, as expected. See the plan [on Phab](https://phabricator.endlessm.com/T26537#726579).

Reply:
Ok, so I will update these changes to depend on malcontent then.